website/src/app/kb/deploy/gateways/readme.mdx

```diff
@@ -115,8 +117,7 @@ Site.
 <Alert color="warning">
   **Note:** Be sure to set a unique `FIREZONE_ID` for each Gateway you deploy.
   This can be any non-empty string and is used to identify Gateways in the
-  portal. This is generated automatically for you when deploying Gateways from
-  the portal.
```

Comment:
I don't see the portal generating `FIREZONE_ID` in the copy-paste code blocks.

Reply:
The Gateway will generate a UUID and store it in `/var` if you don't pass one, so maybe that's what it meant?
https://github.com/firezone/firezone/blob/b64900b513ed0ee7152be0b40a460d6aaec1784e/rust/gateway/src/main.rs#L89-L93

Reply:
We had it required but then removed later, now it's optional - in case your VM doesn't have persistence and you don't want to spam the portal with new gateways on restarts. I think we should recommend setting it somewhere and explain this better.